common/frame_session.go

```diff
@@ -616,16 +617,13 @@ func (fs *FrameSession) onConsoleAPICalled(event *cdpruntime.EventConsoleAPICall
 		}
 		*/
 
-	parsedObjects := make([]any, 0, len(event.Args))
+	parsedObjects := make([]string, 0, len(event.Args))
 	for _, robj := range event.Args {
-		i, err := parseRemoteObject(robj)
-		if err != nil {
-			handleParseRemoteObjectErr(fs.ctx, err, l)
```

Comment:
I'm wondering about the reason for removing error handling.

Reply:
I don't think it needs it for this specific use case since there's nothing that can be done if we return an error. The best course of action is to warn or error on the two cases that you've already highlighted, and then carry on with the processing of the remote object. If we return an error while processing the remote object it will abruptly stop the parsing, but then there's no way to retry it or work with the error to try and solve the parsing issue.